src/modulation/voice_matrix.h

```diff
@@ -134,11 +134,30 @@ enum VoiceModMatrixSource
 
     numVoiceMatrixSources,
 };
+struct VoiceModMatrixSourceMetadata
+{
+    VoiceModMatrixSource id;
+    std::string streamingName;
+    std::string displayName;
+};
+
+const std::vector<VoiceModMatrixSourceMetadata> voiceModMatrixSources = {
+    {VoiceModMatrixSource::vms_none, "vms_none", ""},
+    {VoiceModMatrixSource::vms_LFO1, "vms_lfo1", "LFO1"},
```

Comment:
Minor readability point: Add whitespace.

LFO 1...3
Amp EG
EG 2

Reply:
Ah okay - just saw this. Will do the next time I edit this file.